ui_framework/doc_site/src/views/gallery/gallery.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import {
+  KuiGallery,
+  KuiGalleryButton,
+  KuiGalleryButtonIcon,
+  KuiGalleryButtonImage,
+  KuiGalleryButtonLabel
+} from '../../../../components';
+
+export default () => {
+  /**
+  * These styles are just for demonstration purposes. It is recommended to use
+  * properly named classes instead of CSS properties in production code.
+  */
+  const imageStyle = {
+    backgroundColor: 'lightgray'
+  };
+
+  return (
+    <div>
+      <div className="kuiVerticalRhythm">
+        <div className="kuiSubTitle">
+          Some items
+        </div>
+
+        <KuiGallery>
+          <KuiGalleryButton href="#">
+            <KuiGalleryButtonImage style={imageStyle}/>
+
+            <KuiGalleryButtonLabel>
+              Item A
+            </KuiGalleryButtonLabel>
+
+            <KuiGalleryButtonIcon className="fa-flask"/>
```

Comment:
@cjcenizal Should it be at KuiGalleryButtonIcon automatically? KuiButtonIcon has that on the HTML tag always:
![image](https://user-images.githubusercontent.com/4144816/27068574-133e64e6-5011-11e7-85a7-a5a7e43ede0c.png)

I don't change this now.